zerver/openapi/zulip.yaml

```diff
@@ -6243,12 +6255,24 @@ paths:
             - `oldest`: The oldest message.
             - `first_unread`: The oldest unread message matching the
               query, if any; otherwise, the most recent message.
+            - `date`: The closest message near the searched date. Need to set
+              `anchor_date` argument when using this anchor.
```

Comment:
This should be edited to clearly specify the boundary conditions, like we do for `first_unread`. For example "The oldest message on or after the date indicated in the `anchor_date` parameter, if any; otherwise, the most recent message".

Reply:
done